.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "data"]
 	path = data
-	url = https://github.com/neurodatascience/us_climate_emotions_data
+	url = git@github.com:neurodatascience/us_climate_emotions_data.git
```

Comment:
I wouldn't change to a git URL here since `https`  is needed to automate authentication with a PAT when checking out the private submodule in workflows or during CI/CD (like how we're currently deploying).

Instead, to make local cloning easier, maybe we can update the README instructions for setting up the dev environment with how to modify a local git config to substitute `https` URLs:
```bash
git config --global url.git@github.com:.insteadOf https://github.com/
```
This should cause git to rewrite `https` to `ssh` automatically, so you can use SSH keys.